src/Microsoft.Health.Dicom.Core/Extensions/ExtendedQueryTagEntryExtensions.cs

```diff
@@ -22,7 +23,7 @@ internal static class ExtendedQueryTagEntryExtensions
         /// <param name="extendedQueryTagEntry">The extended query tag entry.</param>
         /// <param name="status"> The status to set on the extended query tag entry.</param>
         /// <returns>Normalize extended query tag entry.</returns>
-        public static ExtendedQueryTagEntry Normalize(this ExtendedQueryTagEntry extendedQueryTagEntry, ExtendedQueryTagStatus status)
+        public static ExtendedQueryTagStoreEntry Normalize(this AddExtendedQueryTagEntry extendedQueryTagEntry, ExtendedQueryTagStatus status)
```

Comment:
It's a bit confusing when Normalize method return a different type .

Reply:
Will rename.